Bronco_Tree/tree.pde

```diff
@@ -17,12 +17,15 @@ class Tree {
     //              a trunk is extended from the bottom until it is within max_dist
     //              of any leaf (essentially, this moves within the vicinity of leaves)
 
-    for (int i = 0; i < 2000; i++) {
+    for (int i = 0; i < 1000; i++) {  // should be 2000 for OG Tree
       leaves.add(new Leaf());
     }
     
+    // for single-leaf tests
+    // leaves.add(new Leaf(new PVector(width/2, 0)));
+    
     // Creating the root branch with dir pointing straight upwards
-    Branch root = new Branch(new PVector(width/2, height), new PVector(0, -1));
+    Branch root = new Branch(new PVector(width/2, height + 5), new PVector(0, -1));
```

Comment:
This is a good way of creating the initial tree, maybe we can have it not grown immediately though? If the user starts with a blank tree, maybe we could have invisible branch points that encourage the growth of a main trunk directly upwards. The first action of the user on a blank screen seems like it would be to attempt to grow a trunk/grow the tree directly upwards